app/src/ui/diff/index.tsx

```diff
@@ -178,6 +221,58 @@ function getLineFilters(
   return { oldLineFilter, newLineFilter }
 }
 
+async function highlightContents(
+  contents: IFileContents,
+  tabSize: number,
+  lineFilters: ILineFilters
+): Promise<IFileTokens> {
+  const { file, oldContents, newContents } = contents
+
+  const [oldTokens, newTokens] = await Promise.all([
+    highlight(
+      oldContents.toString('utf8'),
+      Path.extname(file.oldPath || file.path),
+      tabSize,
+      lineFilters.oldLineFilter
+    ).catch(e => {
+      log.error(`Highlighter worked failed for old contents: ${e}`)
```

Comment:
Shouldn't we pass `e` here instead of interpolating it?

Reply:
Yes we should, nice catch!